python/lsst/sims/GalSimInterface/galSimDetector.py

```diff
@@ -34,18 +35,35 @@ def __init__(self, afwDetector, afwCamera, obs_metadata, epoch):
 
         @param [in] epoch is the epoch in Julian years of the equinox against
         which RA and Dec are measured
+
+        @param [in] photParams is an instantiation of PhotometricParameters
+        (it will contain information about gain, exposure time, etc.)
         """
```

Comment:
I know it's mentioned in the docstring above, but should there still be a line in this docstring for the wcs argument?